kiagnose/internal/config/environment_test.go

```diff
@@ -27,19 +27,18 @@ import (
 	"github.com/kiagnose/kiagnose/kiagnose/internal/config"
 )
 
-func TestConfigMapFullNameShouldSucceed(t *testing.T) {
+func TestCheckupFullNameShouldSucceed(t *testing.T) {
 	goodEnv := map[string]string{
-		config.ConfigMapNamespaceEnvVarName: configMapNamespace,
-		config.ConfigMapNameEnvVarName:      configMapName,
+		config.CheckupNamespaceEnvVarName: checkupKey.Namespace,
+		config.CheckupNameEnvVarName:      checkupKey.Name,
 	}
 
-	namespace, name, err := config.ConfigMapFullName(goodEnv)
+	obtainedKey, err := config.CheckupKeyFromEnv(goodEnv)
 	assert.NoError(t, err)
-	assert.Equal(t, configMapNamespace, namespace)
-	assert.Equal(t, configMapName, name)
+	assert.Equal(t, obtainedKey, checkupKey)
 }
 
-func TestConfigMapFullNameShouldFail(t *testing.T) {
+func TestCheckupFullNameShouldFail(t *testing.T) {
```

Comment:
nit: @qinqon WDYT about
```suggestion
func TestCheckupKeyFromEnvShouldFail(t *testing.T) {
```
?

Reply:
Thank you.